tools/tests/test_airport_darwin_10_9.plist

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Got a preference? There is already a `test_airport.plist` in the tests directory for OS X 10.10 and later.